### ISSUE TITLE
Unable to select Papirus Nord icons with papirus-folders

### ISSUE THREAD

**Issue by sardus-X** [Joshaby/Papirus-Nord#2]
Hi,
I recently installed Your Papirus Nord icons using the install script with sudo (I downloaded the package from the KDE Store). The Papirus Nord icons have been installed in the Papirus icon folders but I cannot select them using papirus-folders command:
`papirus-folders -C "icon-theme" --theme Papirus-Dark`
nor do they appear in the list of available icons using the command:
`papirus-folders -l --theme Papirus-Dark`
I had previously installed papirus-folders before install Your Icon theme.
These are the messages on the terminal during the installation of Your package:
`Welcome to Papirus Custom Folder installer!
        For better usage, run this script using root user!
Papirus-folders already installed!
Appling patch in papirus-folders...
Install.sh: riga 34: patch: comando non trovato
Patch applied!
Instaling icons...
Icons installed!`
How can i fix this problem?
Thanks in advance for Your help

**Comment by Joshaby** [comment]
Hello, for example, use the command papirus-folders -C auroragreen --theme Papirus-Dark for the Aurora Green version! I completely forgot to do a tutorial on how to use the script! I will do this tutorial very soon, I have plans ahead of this!

**Comment by Joshaby** [comment]
Here are the names of the versions that are used in the script: "auroragreen" "auroraorange" "aurorayellow" "frostblue4" "frostblue2" "auroramagenta" "aurorared" "frostblue3" "frostblue" "polarnight3" "polarnight1" "snowstorm1"

**Comment by Joshaby** [comment]
I recommend the icons on the outside, I don't really trust the KDE Store !! And I also recommend downloading the entire pack, because my script only works on it!

**Comment by sardus-X** [comment]
> 
> 
> I recommend the icons on the outside, I don't really trust the KDE Store !! And I also recommend downloading the entire pack, because my script only works on it!
Please, I think I don't understand: what do you mean by "..the icons on the outside"
Do you think it is better to download the package here on Github and reinstall it?

**Comment by sardus-X** [comment]
Please, I think I don't understand: what do you mean by "..the icons on the outside"
Do you think it is better to download the package here on Github and reinstall it?

**Comment by Joshaby** [comment]
I recommend downloading the icons from github, and by them in the folder /usr/share/icons

**Comment by sardus-X** [comment]
Sorry, do you mean extract the "Papirus-Nord-master" folder in / usr / share / icons and then run Your install script?
After this i need to use papirus-folders command to select the icon theme (e.g. aurora orange)?
Update:
i downloaded the package here on github, running Your script with:
`sudo bash install.sh`
gives me the same error:
`Install.sh: riga 34: patch: command not found`

**Comment by sardus-X** [comment]
Solution:
The "patch" command was missing and I installed it with Synaptic.
I re-launched your install script and now the icon themes you created have been added to the papirus-folders theme list.
Evidently the PapirusDevelopmentTeam added recently some icon themes in papirus-folders ("palebrown", "paleorange", "yaru") that your patch removed from papirus-folders theme list, so I re-added them by editing papirus-folders with Nano.
